src/app/components/Metadata/index.jsx

```diff
@@ -92,6 +96,7 @@ Metadata.propTypes = {
   articleSection: string,
   brandName: string.isRequired,
   canonicalLink: string.isRequired,
+  canonicalLinkGb: string.isRequired,
```

Comment:
This works for this specific English language page, however it doesn't scale well to other sites.

We should pass an array of alternateLinks to this front-end component, each with a hrefLang and href value. This array should be looped over for this component. The values should be set in the Metadata Container. 

```
const alternateLinks = [
  {
    hrefLang: 'x-default',
    href: 'https://www.bbc.com/news/articles/c0000000001',
  }
  {
    hrefLang: 'en',
    href: 'https://www.bbc.com/news/articles/c0000000001',
  },
  {
    hrefLang: 'en-gb',
    href: 'https://www.bbc.co.uk/news/articles/c0000000001',
  },
];
```